internal/console/scan.go

```diff
@@ -293,6 +295,7 @@ func setBoundFlags(flagName string, val interface{}, cmd *cobra.Command) {
 	}
 }
 
+//nolint
```

Comment:
complexity/function too long warning? what was it

Reply:
Exactly, the problem in this function is, every time we add a new flag, this function will grow. I couldn't think anyway to reduce the number of lines, do you have any suggestion?